src/utils/SweetAlert.js

```diff
@@ -0,0 +1,44 @@
+import Swal from 'sweetalert2';
+export default {
+  GetAlert: (
+    AlertText,
+    ConfirmButtonText,
+    IsShowCancelButton,
+    AlertIcon,
+    AlertTitle
+  ) =>
+    Swal.fire({
+      title: AlertTitle || 'Alert',
+      text: AlertText,
+      icon: AlertIcon || 'error',
+      showCancelButton: IsShowCancelButton || false,
+      confirmButtonText: ConfirmButtonText,
+      confirmButtonColor: '#3085d6',
+      cancelButtonColor: '#d33',
+    }),
+
+  ConfirmWithAlert: (
+    ConfirmExecute,
```

Comment:
you can do something like this
`ConfirmWithAlert: (
    ConfirmExecute,    option,
  ) =>
    Swal.fire({
      title: 'Are you sure?',
      text: 'Are you sure you want to delete this?',
      icon: AlertIcon,
      showCancelButton: true,
      confirmButtonColor: '#3085d6',
      cancelButtonColor: '#d33',
      confirmButtonText: 'Yes, delete it!',
...option
    }).then((result) => {
      if (result.isConfirmed) {
        ConfirmExecute();
        Swal.fire('Deleted!', 'The deletion was successful', 'success');
      } else {
        Swal.fire('oops!', 'Your not deleted', 'error');
      }
    }),
};`

then u can call the function like this 
foo(()=>{},{text:'are yo sure again})

good job .